dev-docs/bidders/ogury.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: bidder
+title: ogury
+description: Ogury Bidder Adapter
+biddercode: ogury
+gdpr_supported: true
+gvl_id: 45
```

Comment:
GVL 45 is Smart Ad Server. Yours appears to be 31. Please confirm and update.

Reply:
Yes I confirm, thank you & sorry for the mistake